### PR TITLE
Quick steps for generating hyperkube image

### PR DESCRIPTION
**Pull request by kkmsft**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

/kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Quick steps for generating hyperkube image

During iterative development process, its effective to just build only required binaries for the specific architecture and then generate the hyperkube image from it. This PR adds two simple steps to perform those operations with the latest script based hyperkube.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cluster/images/hyperkube/README.md [hunk 1]
```diff
@@ -6,6 +6,20 @@
 
 #### How to release by hand
 
+##### Quick steps
```
Comment (dims) [inline cluster/images/hyperkube/README.md]: Can we please make it clear that this is for a single container image for a specific architecture for development/CI scenarios?

Comment (kkmsft) [reply cluster/images/hyperkube/README.md]: Thank you, @dims . Have refactored to bring out the development scenario aspect. PTAL.

**Comment by dims** [comment]
/ok-to-test
/important-soon
/approve
/lgtm

@kkmsft thanks KK

**Pull request merged**